lib/components/split-pane.js

```diff
@@ -18,6 +20,28 @@ export default class SplitPane extends PureComponent {
     }
   }
 
+  setupPanes(ev) {
+    this.panes = Array.from(ev.target.parentNode.childNodes);
+    this.paneIndex = this.panes.indexOf(ev.target);
+    this.paneIndex -= Math.ceil(this.paneIndex / 2);
+  }
+
+  handleAutoResize(ev) {
+    ev.preventDefault();
+
+    this.setupPanes(ev);
+
+    const sizes_ = this.getSizes();
+    sizes_[this.paneIndex] = 0;
+    sizes_[this.paneIndex + 1] = 0;
+
+    const availableWidth = 1 - _.sum(sizes_);
```

Comment:
I prefer using map/filter/reduce when it is possible :)

`const availableWidth = sizes_.reduce((total, size) => total - size, 1);`